page-listing.php

```diff
@@ -0,0 +1,274 @@
+<?php
+/**
+ * Template Name: Listing
+ *
+ * @package Hale
+ * @copyright Ministry of Justice
+ * @version 2.0
+ */
+
+get_header();
+
+//Get Search text Value
+
+$listing_search_text = '';
+$listing_active_filters = [];
+
+if (get_query_var('listing_search')) {
+
+    $listing_search_text = sanitize_text_field(get_query_var('listing_search'));
+    $search_text_HTML = esc_html($listing_search_text);
+    $search_text_HTML = str_replace('\\', '', $search_text_HTML); // kill backslashes
+}
+
+while (have_posts()) :
+    the_post();
+
+    //Check if documents are restricted
+    $restrict_documents = get_post_meta(get_the_ID(), 'restrict_documents', true);
+
+    ?>
+
+    <div id="primary" class="govuk-grid-column-full-from-desktop">
+        <h1 class="govuk-heading-xl">
+            <?php echo get_the_title(); ?>
+        </h1>
+
+        <?php
+        // Page body content
+        get_template_part('template-parts/content', 'page');
+        ?>
+
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-one-third">
+                <div class="listing-search-section">
+                    <div class="listing-search-form">
+                        <form action="<?php echo get_permalink(); ?>" method="GET">
+                            <label for="listing-search-field" class="govuk-visually-hidden">
+                                <?php _e('Search', 'hale'); ?>
+                            </label>
+                            <input class="govuk-input" id="listing-search-field" name="listing_search"
+                                   value="<?php echo $search_text_HTML; ?>" type="search"
+                                   placeholder="<?php _e('Search', 'hale'); ?>">
+
+                            <?php
+
+                            $listing_filters = get_field('listing_filters');
+
+                            if(!empty($listing_filters) && is_array($listing_filters)) { ?>
+
+                                    <p>Filters</p>
```

Comment:
The "Filters" title displaying is based on the $listing_filters being populated in the backend regardless of whether the taxonomy has been assigned to a post or not. Meaning that it is possible to have the Filter title appear with no filter boxes below or posts showing. Probably a post deploy thing to review.